<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>220</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>좌표계 변환</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="1" colspan="7">
    <widget class="QComboBox" name="comboBox_TargetCoordinates"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>저장 경로 :</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="7">
    <widget class="QComboBox" name="comboBox_Layers"/>
   </item>
   <item row="3" column="7">
    <widget class="QPushButton" name="pushButton_SetSaveFilePath">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="5" column="7">
    <widget class="QPushButton" name="pushButton_Cancel">
     <property name="text">
      <string>취소</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="7">
    <widget class="QLineEdit" name="lineEdit_CurrentCRS">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>대상 좌표계 :</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>레이어 :</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>현재 좌표계 :</string>
     </property>
    </widget>
   </item>
   <item row="5" column="6">
    <widget class="QPushButton" name="pushButton_ConvertCRS">
     <property name="text">
      <string>변환</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="6">
    <widget class="QLineEdit" name="lineEdit_SaveFilePath"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
